ui/src/timeMachine/reducers/index.test.ts

```diff
@@ -25,40 +25,33 @@ import {
 // Utils
 import {createView} from 'src/shared/utils/view'
 
-// Constants
-import {TimeMachineEnum} from 'src/timeMachine/constants'
-
 // Types
-import {TimeMachineTab} from 'src/types/timeMachine'
 import {DashboardDraftQuery, QueryViewProperties} from 'src/types/dashboards'
 import {selectAggregateWindow} from '../actions/queryBuilder'
 
 describe('timeMachinesReducer', () => {
```

Comment:
We should just delete this test... I've never seen it catch regressions and have had to change it so many times while refactoring.

Reply:
deleted.